docs/operating-scylla/procedures/cluster-management/rebuild-node.rst

```diff
@@ -1,19 +1,25 @@
 ============================================
-Rebuild a Node After Losing the Data Volume
+Recover a Node After Losing the Data Volume
 ============================================
 
-When running in Scylla on EC2, it is recommended to use i3 type instances, storing the data on fast, ephemeral SSD drives.
-Stopping the node and starting it for whatever reason means the data on the node is lost.
-This data loss applies not only to i3 type instances but to the i2 type as well.
+When running in Scylla on EC2, it is recommended to use instances with fast, ephemeral SSD drives.
+Stopping the instances and starting it for whatever reason means the data on the node is lost.
+
 The good news is Scylla is a HA database, and replicas of the data are stored on additional nodes.
-This is also why it's highly recommended to use a replication factor of at least three per Data Center (for example, 1 DC, RF = 3, 2 DCs RF = 6).
+This is also why it's highly recommended to use a replication factor of at least three per Data Center, for example, ``replication = {'class': 'NetworkTopologyStrategy', 'DC1' : 3, 'DC2' : 3}``.
 
-To recover the data and rebuild the node, follow this procedure:
+The following procedure is a particular use case of :doc:`Replace a Dead Node in a ScyllaDB Cluster Procedure </operating-scylla/procedures/cluster-management/replace-dead-node>`, when a node replaces itself using its previous identity (Host ID) and IP address.
 
 #. Open the ``/etc/scylla/scylla.yaml`` file.
 
 #. Add, if not present, else edit, the ``replace_node_first_boot`` parameter and change it to the
    Host ID of the node before it restarted.
+
+#. If you know of dead nodes in the cluster, for example, from :doc:`nodetool status </operating-scylla/nodetool-commands/status>`, you can use :doc:`ignore_dead_nodes_for_replace </reference/configuration-parameters>` to ignore these nodes for the replace.
```

Comment:
I'm not a native English speaker but to my ears, "those" rhymes better than "these" here.

@annastuchlik what do you think?

Reply:
https://dictionary.cambridge.org/grammar/british-grammar/this-that-these-those

> We use that and those most commonly to point to things and people which are not easy to identify in a situation. They are often more distant from the speaker, and sometimes closer to the listener:
>
> Sometimes they are not visible to either the speaker or listener:

The dead nodes are distant and vague, so they should be referred to as "those", not "these".